metrics/prometheus.go

```diff
@@ -902,6 +902,32 @@ const (
 	LabelImage = "image"
 )
 
+// WhiteListedContainerLabels returns a ContainerLabelsFunc that exports the
+// container name, first alias, image name as well as white listed label
+// values.
+func WhiteListedContainerLabels(whiteList []string) func(container *info.ContainerInfo) map[string]string {
```

Comment:
I don't think we need seperate `WhiteListedContainerLabels` and `BaseContainerLabels`.  BaseContainerLabels is just the case where `WhitelistedContainerLabels` is passed an empty list.